### ISSUE TITLE
Can't add cursor above or below for multi-cursor when created with task

### ISSUE THREAD

**Issue by xconverge** [VSCodeVim/Vim#860]
---

Please _thumbs-up_ 👍 this issue if it personally affects you! You can do this by clicking on the emoji-face on the top right of this post. Issues with more thumbs-up will be prioritized.

---
### What did you do?

Tried to select add cursor above/below from the tasks list (ctrl + alt + arrow works)
### What did you expect to happen?

Add a cursor above or below that stayed active
### What happened instead?

When using the task from ctrl+p menu, the cursor appears to be added, but as soon as I navigate, the 2nd cursor disappears and does not work with any commands, it is back to 1 cursor
### Technical details:
- VSCode Version: master
- VsCodeVim Version: 1.5.3
- OS: Win10x64

**Comment by xconverge** [comment]
Ok so I quit everything and restarted and now I can use ctrl + alt +arrow, however the task does the same thing still, creates the cursor but if I press i or navigate it disappears

**Comment by xconverge** [comment]
Looks like we are not running the following when we invoke Add Cursor Above from the task list instead of using the actual keyboard shortcut that we intercept.

```
vimState.allCursors = await allowVSCodeToPropagateCursorUpdatesAndReturnThem();
```

**Comment by xconverge** [comment]
fixed with #1434 on master
